gourmet/plugins/browse_recipes/browser.py

```diff
@@ -67,12 +67,12 @@ def get_pixbuf (self, attr,val):
             tbl = self.rd.recipe_table.join(self.rd.categories_table)
             col = self.rd.categories_table.c.category
             if hasattr(self,'category_images'):
-                stment = and_(col==val,self.rd.recipe_table.c.image!=None,
-                              self.rd.recipe_table.c.image!='',
-                              not_(self.rd.recipe_table.c.title.in_(self.category_images))
-                              )
+                stment = and_(col == val.encode(), self.rd.recipe_table.c.image != None,
+                              self.rd.recipe_table.c.image != ''.encode(),
```

Comment:
you can put `bytes()` instead of `''.encode()'. It's an empty bytestream after all.

Reply:
True, thanks!